packages/babel-generator/src/index.js

```diff
@@ -58,7 +58,6 @@ function normalizeOptions(code, opts, tokens): Format {
     compact: opts.compact,
     minified: opts.minified,
     concise: opts.concise,
-    quotes: opts.quotes || findCommonStringDelimiter(code, tokens),
```

Comment:
Yeah, I guess we never really decided if we wanted to default to `double` or infer. The more I think about it, I think infer is better option?